openreview/venue_request/venue_request.py

```diff
@@ -508,6 +522,7 @@ def setup_comment_stage(self):
                 'description': 'Should the PCs receive an email for each official comment made in the venue? Default is "No, do not email PCs for each official comment in the venue"',
                 'value-radio': [
                     'Yes, email PCs for each official comment made in the venue',
+                    'Yes, email PCs for each direct official comment made in the venue',
```

Comment:
```suggestion
                    'Yes, email PCs only for private official comments made in the venue (comments visible only to Program Chairs and Senior Area Chairs, if applicable)',
```

Reply:
I know this is kind of long and I am open to other ideas, but I don't think just saying "private" or "direct" will be clear enough for PCs

Reply:
I like this option